keymap.cson

```diff
@@ -26,6 +26,8 @@
 
 # window/pane navigation
 'body, atom-text-editor':
+  'shift-cmd-y': 'pane:show-next-item'
+  'shift-cmd-h': 'pane:show-previous-item'
```

Comment:
:thought_balloon: I wanted to use H and L, but L is taken. H was free and Y was near by and free. these are not so vimmy but they play well w/ the focus previous/next pane as below.